src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_NUM = 7;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random randomNum = new Random();
+
+    public Figure getRandomFigure() {
+        int randomFigureNumber = randomNum.nextInt(5);
+        switch (randomFigureNumber) {
+            case 0:
+                return getRandomIsoscelesTrapezoid();
+            case 1:
+                return getRandomRectangle();
+            case 2:
+                return getRandomRightTriangle();
+            case 3:
+                return getRandomSquare();
+            case 4:
+                return getRandomCircle();
+            default:
+                return null;
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", 10);
```

Comment:
```suggestion
        return new Circle(DEFAULT_COLOR, DEFAULT_FIGURE);
```
lets create constants